libgdf/src/transpose.cu

```diff
@@ -0,0 +1,49 @@
+/* Copyright 2018 NVIDIA Corporation.  All rights reserved. */
+
+#include "gdf_table.cuh"
+#include <gdf/gdf.h>
+#include <thrust/for_each.h>
+#include <thrust/iterator/counting_iterator.h>
+#include <memory>
+#include <stdio.h>
+
+gdf_error gdf_transpose(size_t ncols,
```

Comment:
Documentation as laid out in template here: https://github.com/rapidsai/cudf/blob/master/libgdf/src/example_documentation.cpp

Reply:
Do i need to add the documentation in both header and source?